airflow/providers/google/cloud/hooks/bigtable.py

```diff
@@ -94,7 +94,7 @@ def delete_instance(self, instance_id: str, project_id: Optional[str] = None) ->
         if instance:
             instance.delete()
         else:
-            self.log.info("The instance '%s' does not exist in project '%s'. Exiting", instance_id,
+            self.log.warn("The instance '%s' does not exist in project '%s'. Exiting", instance_id,
```

Comment:
Here is the pylint problem because the indentation is incorrect ..

Reply:
Thanks @mik-laj , I booted up breeze again and found a few such issues, I think all are fixed now.